GlyssenEngine/Quote/ParserCharacterRepository.cs

```diff
@@ -45,25 +45,30 @@ public ParserCharacterRepository(ICharacterVerseRepository cvInfo, ReferenceText
 				"narrator overrides. These should be applied only after user has done disambiguation.");
 			var result = m_cvInfo.GetCharacters(bookId, chapter, verses, versification,
 				includeAlternatesAndRareQuotes, includeNarratorOverrides);
-			var hypotheticalsToReplace = result.Where(cv =>
-				{
-					if (cv.QuoteType == QuoteType.Hypothetical)
-					{
-						var refTextBook = m_referenceText.GetBook(bookId);
-						if (refTextBook != null)
-							return !refTextBook.GetBlocksForVerse(chapter,
-								verses.First().StartVerse, verses.Last().EndVerse).Any(b => b.CharacterId == cv.Character);
-						// REVIEW: Should we replace hypotheticals if there is no reference text for this book?
-					}
-					return false;
-				}).Select(c => c.Character).ToList();
-			if (hypotheticalsToReplace.Any())
+			// m_referenceText will never be null for Glyssen users, so the test for null will not effect Glyssen.
+			// At this time, Vessel users never have a reference text, so this test prevents an exception being thrown when the GlyssenEngine is called by Vessel.
```

Comment:
Just remove " when the GlyssenEngine is called by Vessel"

Reply:
OK sorry, done.